reflex.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: reflex
+  version: 0.2.4
+  epoch: 0
+  description: "Web apps in pure Python"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - python3
+      # Reflex shells out to these these tools during init
+      - bash
+      - wget
+      - curl
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
```

Comment:
super nit, wonder if we should strive for alphabetizing packages so it's easy to see what's in there. Maybe it's just me.

Reply:
That seems like a useful thing to add to https://github.com/chainguard-dev/yam and apply across this repo, if you're feeling so inclined.

Personally I'd love to take advantage of our knowledge of the dep graph to trim away redundant deps. 🙏